src/platforms/common/px4_work_queue/test/wqueue_scheduled_test.h

```diff
@@ -0,0 +1,24 @@
+
+#pragma once
+
+#include <px4_app.h>
+#include <px4_work_queue/ScheduledWorkItem.hpp>
```

Comment:
I see `.h` and `.hpp` mixed? Can we agree on one?

Reply:
I use hpp for things that can only be used with c++ (classes).

Reply:
Ok.

Reply:
Can we come to a general agreement?

Reply:
We could talk about it on the dev call to see if there are more opinions. I have no strong opinion.